README.md

```diff
@@ -35,9 +35,23 @@ check the `examples` subdirectory.
 
 ## Install
 
+Install it via go get
 ```
 go get github.com/owulveryck/onnx-go
 ```
+Or add it in go.mod
+
+```
+module myproject.org/package
+
+go 1.12
+
+require (
+        github.com/google/flatbuffers v1.11.0 // indirect
+        github.com/owulveryck/onnx-go v0.0.0-20190605120756-5af9985
```

Comment:
I know nothing about modules, does this version change at each commit?

Reply:
isn't it a way to generate it "automatically"?

Reply:
User can generate by `go mod init`, you right.
I will just add the project support go modules.